apps/web/src/components/Header/Header.tsx

```diff
@@ -32,8 +32,8 @@ const Header = ({ width }: HeaderProps) => {
   }
   return (
     <header
-      data-h2-background-color="base(foreground)"
-      data-h2-border-bottom="base(1px solid black.20) base:dark(1px solid white.20)"
+      data-h2-background-color="base(foreground) base:dark(white)"
```

Comment:
Yeah, the hero uses a much darker black compared to the rest of the site. It's the only instance where this happens, but because it's part of the original design, I haven't changed it.

Reply:
Oh sorry, I realized I didn't say what I was referring to! At least to my eyes, the header background looks different than the buttons and the content background to me.

But, I looked and I think it is just my eyes being bad 😵‍💫

Reply:
Oh, gotcha - no your eyes are fine - in this case, the header and footer are using a darker color, and the buttons are using `foreground`. The rest of the general contents sections are using `background`.